C.py

```diff
@@ -0,0 +1,2 @@
+def byby():
```

Comment:
```suggestion
def byebye():
```
byby→byebyeに修正お願いします。